### ISSUE TITLE
[UI] Tabbed Schedule View, collapsing browser size (responsive) jumps tab to another day to that already selected

### ISSUE THREAD

**Issue by astevens93** [netmix/radio-station#229]
**Describe the bug**
On Tabbed Schedule View, collapsing browser size (responsive) jumps tab to another day to that already selected.

**To Reproduce**
1. Load up tabbed schedule in web browser
2. Select a day
3. Reduce browser size
4. Watch selected day jump to another day automatically, therefore moving the user off the day already selected.

**Expected behavior**
To stay on the selected day.

**Desktop (please complete the following information):**
 - OS: Mac OS X
 - Browser Chrome
 - Version 83

**Comment by astevens93** [comment]
P.s I also notice this upon scrolling on IOS

A link to the page I'm experiencing issues on. I have only amended the CSS to style to my desires:
https://www.hcrfm.co.uk/schedule-new/

**Comment by tonyzeoli** [comment]
Sorry, I could not replicate this issue on either your site or our radiostationdemo.com. The day did not switch when pulling in the browser from the corner. I'm using Chrome for Mac.

I'll leave this issue open for discussion for one more round, but I don't see the issue. Can you try someone else's machine? Can you try a different browser?

**Comment by astevens93** [comment]
Hi Tony, thanks for coming back to me. I've recorded the issue happening on Chrome for Mac. It also happens on Safari.

2 Brief videos on my site, and radiostationdemo.com
https://we.tl/t-OeYO6mM7xQ

If I select Saturday or Sunday like in the example, it eventually jumps back to a weekday.

I re-created the issue each time when the following days are active:

Monday jumps to Tuesday
Current day not affected (today is Tuesday)
Wednesday not affected
Thursday not affected
Friday jumps to Thursday
Saturday & Sunday jump to Thursday

**Comment by tonyzeoli** [comment]
So sorry, you're right. I didn't follow your instructions, but when I did I figured out that yes, in responsive the tabs are switching incorrectly to a different tab than was selected. We'll fix this and push it in the next release.

[Screen Recording 2020-06-23 at 16.19.36.mp4.zip](https://github.com/netmix/radio-station/files/4820393/Screen.Recording.2020-06-23.at.16.19.36.mp4.zip)

**Comment by tonyzeoli** [comment]
@majick777 We've got an issue here. I've confirmed the issue where tabs switch to another day in responsive states when pulling in the browser from the corner wide to narrow view. I've added this to our 2.3.2 project.

**Comment by astevens93** [comment]
Thank you so much, your attention to this project really is appreciated.

**Comment by tonyzeoli** [comment]
Thanks. You can assist by leaving a positive review on the Wordpress.org
Reviews page.

You can also add your station to our directory for free til the end of July
(we're changing the end date). Check Netmix.com for details!


Best regards,


*Tony Zeoli*
[image: Digital Strategy Works logo] [image: netmix logo]
Digital Strategy Works + Netmix
Web Development, Media & Music
✉ tonyzeoli@netmix.com | az@digitalstrategyworks.com
☎ 917.705.4700
<http://www.facebook.com/digitalstrategyworks>
<http://www.twitter.com/dswks> <http://www.facebook.com/netmix>
<http://www.twitter.com/netmix>
Create your own email signature
<https://www.wisestamp.com/signature-in-email/?utm_source=promotion&utm_medium=signature&utm_campaign=create_your_own&srcid=5277383900790784>

**Comment by majick777** [comment]
@astevens93 Okay I've checked this out and it's a good point, this isn't the best behaviour. I do recall adding this because I didn't want the user to get confused what day they are viewing when the currently selected day tab is no longer showing. But changing the selected day is not the right solution to that.

Instead, I've changed this to display a "Viewing: {day}" above the Shows list when the currently displayed day tab is no longer visible. This should only happen on window resize or moving left/right anyway, as the current day tab is the default view on pageload.

Added to 2.3.1.6 development version for 2.3.2 release.
https://netmix.com/radio-station/docs/FAQ/#how-do-i-install-the-latest-development-version-for-testing
